<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhongTexturedMaterialWidgetBase</class>
 <widget class="QWidget" name="PhongTexturedMaterialWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QgsDoubleSpinBox" name="textureScaleSpinBox">
     <property name="suffix">
      <string> %</string>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QgsDoubleSpinBox" name="spinShininess">
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QgsColorButton" name="btnSpecular">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QgsImageSourceLineEdit" name="textureFile" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblSpecular">
     <property name="toolTip">
      <string>Color of light reflecting from smooth surfaces.</string>
     </property>
     <property name="text">
      <string>Specular</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblTextureRotation">
     <property name="text">
      <string>Texture rotation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblAmbient">
     <property name="toolTip">
      <string>Color of light that is scattered around the entire scene.</string>
     </property>
     <property name="text">
      <string>Ambient</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblTextureScale">
     <property name="text">
      <string>Texture scale</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblShininess">
     <property name="toolTip">
      <string>How shiny smooth surfaces are.</string>
     </property>
     <property name="text">
      <string>Shininess</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsColorButton" name="btnAmbient">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblTextureScale_2">
     <property name="text">
      <string>Diffuse texture</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QgsDoubleSpinBox" name="textureRotationSpinBox">
     <property name="suffix">
      <string> Â°</string>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblOpacity">
     <property name="text">
      <string>Opacity</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QgsOpacityWidget" name="mOpacityWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsImageSourceLineEdit</class>
   <extends>QWidget</extends>
   <header>qgsfilecontentsourcelineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnAmbient</tabstop>
  <tabstop>btnSpecular</tabstop>
  <tabstop>spinShininess</tabstop>
  <tabstop>textureScaleSpinBox</tabstop>
  <tabstop>textureRotationSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
